treenode/admin.py

```diff
@@ -6,6 +6,17 @@
 from .forms import TreeNodeForm
 
 
+class TreeNodeModelAdminInline(admin.TabularInline):
+    model, instance = None, None
+
+    def get_formset(self, request, obj=None, **kwargs):
+        self.instance = obj
+        return super().get_formset(request, obj, **kwargs)
```

Comment:
This package supports python 2.7, so the super call must receive the 2 arguments.